cluster-operator/src/test/java/io/strimzi/operator/cluster/model/CertUtilsTest.java

```diff
@@ -106,4 +113,132 @@ public void testExistingCertificatesDiffer()   {
         assertThat(CertUtils.doExistingCertificatesDiffer(defaultSecret, changedScaleUpSecret), is(true));
         assertThat(CertUtils.doExistingCertificatesDiffer(defaultSecret, changedScaleDownSecret), is(true));
     }
+
```

Comment:
Should there be a test for adding multiple volumes/volume mounts with the same secret name?

Reply:
@katheris The tests do have the same Secret twice -> the `first-certificate` Secret is used in both certificates. So that seems to be covered? Or maybe I did not understand what exactly you meant?

Reply:
Ah I somehow missed that, my mistake!